app/views/setup/_registry.html.slim

```diff
@@ -0,0 +1,22 @@
+= form.fields_for "insecure_registries[]", [] do |f|
+  - certificate_docker_registry = Certificate.find(registry.certifiable_id).certificate rescue ""
+  .row.registry-entry
+    .form-group.col-sm-4
+      = f.label :docker_registry_name, "Registry name"
+      = f.text_field :docker_registry_name, class: "form-control", placeholder: "myregistrydomain.com:5000", value: registry.name, readonly: !registry.name.blank?
+
+      .checkbox
+        label
+          = f.check_box :docker_registry_mirror, { multiple: true, checked: registry.mirror }, "true", nil
+          | Enable mirror
+
+      button.btn.btn-danger.btn-remove-registry type="button"
+        span.fa.fa-minus aria-hidden="true"
+        |  Remove registry
+      button.btn.btn-success.btn-add-registry type="button"
+        span.fa.fa-plus aria-hidden="true"
+        |  Add another registry
+
+    .form-group.col-sm-8
+      = f.label :docker_registry_certificate_name, "Certificate"
+      = f.text_area :docker_registry_certificate, class: "form-control", placeholder: "TODO example of certificate in helper", value: certificate_docker_registry, cols: 80, rows: 18, readonly: !registry.certifiable_id.blank?
```

Comment:
@inercia regarding the TODO, should we add a real example, or just some information like "paste a valid certificate here"

Reply:
I would say an example of a certificate to let the user know what he really needs to paste there.

Since it's gonna be a big string, put in a helper function and call it there.

Reply:
@inercia and i think this might be a bit confusing in the end to have an example in there, would you be fine as well with just an explanation?

Reply:
I personally prefer an example but I guess an explanation is fine too.